### PR TITLE
add --conversation_id, normalize method, do_action

### PR DESCRIPTION
**Pull request by pjq**

### CODE REVIEW

File: xiaogpt.py [hunk 1]
```diff
@@ -165,7 +178,15 @@ def run_forever(self):
         default="LX06",
         help="小爱 hardware",
     )
+
+    parser.add_argument(
+        "--conversation_id",
+        dest="conversation_id",
+        type=str,
+        default="c379851d-fbd4-479e-a716-dddc379fbxxx",
```
Comment (yihong0618) [inline xiaogpt.py:186]: let us make default empty string.
then user do not need to give a conversation_id also work

File: xiaogpt.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def parse_cookie_string(cookie_string):
 
 
 class MiGPT:
-    def __init__(self, hardware):
+    def __init__(self, hardware, conversation_id):
```
Comment (yihong0618) [inline xiaogpt.py:36]: conversation_id = "" default value

**Review by yihong0618 — CHANGES_REQUESTED** [review]

**Comment by wutaijieing** [comment]
conversation_id 需要如何获取？刚接触，请教下

File: xiaogpt.py (hunk 1 above)
Comment (pjq) [reply xiaogpt.py]: done

File: xiaogpt.py (hunk 2 above)
Comment (pjq) [reply xiaogpt.py]: done

**Comment by yihong0618** [comment]
我稍微改动了下，感谢。

**Pull request merged**